msteams-platform/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md

```diff
@@ -1,228 +1,192 @@
+The information that you submit to [Partner Center](https://partner.microsoft.com)&#8212;including your name, descriptions, icons, and images&#8212;becomes the Microsoft Teams store and Microsoft AppSource listing for your app.
 
-> [!NOTE]
-> Make sure your app information follows our [AppSource guidance for creating an effective store listing](/office/dev/store/create-effective-office-store-listings).
+A store listing may be someone's first impression of your app. Increase your installations with a listing that effectively conveys your app's benefits, functionality, and brand.
 
-## App name
+## 1. Come up with a short name
 
-> [!div class="checklist"]
->
-> * An app's name plays a critical role in how users discover it in the AppSource app store. Your app's short name is displayed on the details page.
->* The app name should reflect your app without any reference to Microsoft or Microsoft products.
->
+Your app's name (specifically, its [*short name*](~/resources/schema/manifest-schema.md#name)) plays a crucial role in how users discover it in the store.
 
-> **Note**: If your app is an official partnership with Microsoft, then the name of the third-party app needs to be first, e.g., *Salesforce Connector for Microsoft Teams*.
+The following example highlights where an app's short name displays in a store listing.
 
-> [!div class="checklist"]
->
->* Use these resources for guidance:
+:::image type="content" source="../../../../assets/images/store-detail-page/AppName-02.png" alt-text="Example screenshot highlights where an app's short name displays in a store listing.":::
 
-* [App name guide](#app-name)
-* [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)
+### Best practices for names
 
-**Do's:**
+**Do:**
 
 * Choose a simple, memorable name that hints at what your app does.
 * Be distinctive.
-* If needed, use Microsoft 365 references instead of Office 365.
-
-**Don'ts:**
+* If needed, reference "Microsoft 365" instead of "Office 365".
+* Avoid typos and grammatical errors.
 
-* Don't omit spaces, have an incorrect case, or contain language errors in the app name.
-* Don't use generic terms or names similar to existing apps.
-* Don't use "Teams", "Microsoft", existing/upcoming Microsoft product names, or  "app" in your app name.
-* Don't use parenthesis to include Microsoft products, e.g., *Your-App-Name (for Microsoft Teams)*.
+**Don't:**
 
-![App name store view](../../../../assets/images/store-detail-page/AppName-02.png)
+* Use profane or derogatory terms.
+* Use racially or culturally insensitive language.
+* Use generic terms or names similar to existing apps.
+* Include "Teams", "Microsoft", existing/upcoming Microsoft product names, or  "app" in the name.
```

Comment:
Conflicting info about if you can use Microsoft product names or not in your app name (i.e., short name). Can someone please clarify?